### DISCUSSION TITLE
Webpack-dev-server help

### DISCUSSION THREAD

**Discussion by dunmorogher** [webpack/webpack#14828] [Q&A] [answered]
Hi There!
I am a little new to ReactJS (ts, webpack, etc....), and just encountered one issue - I am currently using `webpack: 5.64.3` and `webpack-dev-server: 4.5.0`. I want to start webpack-dev-server in my local env. 
I wrote the devServer config below in my "webpack.config.js" file
```js
devServer: {
    static: {
        directory: path.join(__dirname, 'public')
    },
    compress: true,
    open: false,
    port: 8080,
    client: {
        progress: true,
        logging: 'info'
    },
    liveReload: false,
    hot: true,
    // host: '0.0.0.0',
}
```
And I use the command `webpack serve --config ./webpack.config.js` to start webpack-dev-server. And finally the output from CMD window is

![圖片](https://user-images.githubusercontent.com/41850822/143430098-c4447579-1178-4751-a963-7dc54f7f3bd9.png)

But at the end I tried to access http://localhost:8080 the browser was pending on... it seemed that I couldn't be able to access the app. The icon from browser's tab is whirling all the time...

![圖片](https://user-images.githubusercontent.com/41850822/143430723-410b1b11-49a9-4768-819b-37b46435a5d7.png)

Appreciate your helps, thanks in advance!

**Comment by alexander-akait** [comment, marked as answer]
What is bundle size?

**Reply by dunmorogher** [reply]
Hi! Sorry bundle size? I couldn't see any bundle file...
Below are my configuration codes about `output`. I am not sure for my case if the process is still staying on compiling stage?
```js
module.exports = {
    mode: 'development',
    entry: './src/index.tsx',
    output: {
        publicPath: '/',
        filename: 'bundle.js',
        path: path.resolve(__dirname, 'dist')
    }
    ...
}
```

**Reply by alexander-akait** [reply]
Based on output you have very big file, it can be problem in browser, can you run webpack without dev server

**Reply by dunmorogher** [reply]
Thanks. Because I just want to see the UI of my app from browser... I don't know why the file would be large? Would be that I didn't configure the "webpack.config.js" correctly? Sorry I am a little new and can you point me out with the optimization correction for that file?😅

**Reply by alexander-akait** [reply]
Can you run webpack wihout dev server?

**Reply by dunmorogher** [reply]
I just did that, and it seemed pending on this state all the time. It looks like Webpack was compiling many of my project's files? I think I didn't have many of those files needed to be processed...?

![圖片](https://user-images.githubusercontent.com/41850822/143436078-18c82d3a-6f67-4916-98f1-e3ca4e15e79f.png)

If somewhere of my configuration is wrong?

**Reply by alexander-akait** [reply]
Check your imports

**Reply by dunmorogher** [reply]
Hi, thank you so much! Yes, I just checked out my imports of some files and just commented these two lines in one file
```js
// import '../css/antd.min.css'
// import '../js/antd.min.js'
```
And then webpack command worked well... I am just wondering if this is reduplicated imports against what I did at line 4?

![圖片](https://user-images.githubusercontent.com/41850822/143442841-2d19e098-3110-44cf-87df-3c75329ea7b6.png)

**Reply by alexander-akait** [reply]
Yep, looks like you have problem in CSS and output error/warning is very long, if you will wait longer you will see it, issue https://github.com/webpack/webpack/issues/14678, you can comment this block and it should be faster

**Reply by dunmorogher** [reply]
OK thanks, I understand. Will check again by my own! :-)
